lib/Common/Memory/RecyclerPointers.h

```diff
@@ -145,10 +145,14 @@ template <>
 struct _ArrayItemWriteBarrierPolicy<_write_barrier_policy>
     { typedef _write_barrier_policy Policy; };
 
+template <class T, int N> // in case calling with type WriteBarrierPtr<void> [N]
```

Comment:
Is this used/effective anywhere? If we have a `WriteBarrierPtr<T>[]` parameter, it should match `WriteBarrierPtr<T>*`.